apps/prairielearn/assets/scripts/instructorAssessmentManualGradingInstanceQuestion.js

```diff
@@ -373,34 +404,43 @@ function updatePointsView(sourceInput) {
 
     form
       .querySelectorAll('.js-auto-score-value-input-points')
-      .forEach((input) => input !== sourceInput && (input.value = auto_points));
+      .forEach((input) => input !== sourceInput && (input.value = `${auto_points}`));
```

Comment:
Nit: I'd be more inclined to write this as `String(auto_points)` to make the intent clearer. Ditto elsewhere.

Reply:
I was too, until I learned the performance is  5x worse. https://stackoverflow.com/a/51503252

Reply:
This can't _posssibly_ make a difference for us.  If we ever get a rubric that has even remotely enough rubric items for this to matter, we'll have far larger problems on our hands.

Reply:
Makes sense 🤣 . I'll admit I also prefer the `${}` syntax for brevity. In the typescript PR i'll change these to `String`.